Tasks/task1.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+
+<head>
+    <title>Task 1</title>
+</head>
+
+<body>
+    <h1>CookAsistant</h1>
+    <input id="input" placeholder="Product Name" />
+    <button onclick="add()">Add Product</button>
+
+    <h2>Products</h2>
+    <ul id="list"></ul>
+
+    <script>
+        let products = []
+        const list = document.getElementById("list")
+        const input = document.getElementById("input")
+
+        // Refresh products list
+        function refresh() {
+            list.innerHTML = products.map(({ id, name }) =>
+                `<li>
+                    <span>${name}</span>
+                    <button onclick="remove(${id})">x</button>
+                </li>`
+            ).join("")
+        }
+        function add() {
+            products.push({ id: Math.random(), name: input.value, })
+            input.value = ""
+            refresh()
```

Comment:
As mentioned before, refresh() function here can be extracted outside add() function for reusability